scylla-cdc-driver3/src/main/java/com/scylladb/cdc/cql/driver3/Driver3WorkerCQL.java

```diff
@@ -127,9 +127,15 @@ public void onFailure(Throwable t) {
             } else {
                 Row row = rs.one();
                 if (schema == null) {
-                    schema = new Driver3SchemaBuilder()
-                            .withClusterMetadata(session.getCluster().getMetadata())
-                            .withRow(row).build();
+                    try {
+                        schema = new Driver3SchemaBuilder()
+                                .withRow(row)
+                                .withClusterMetadata(session.getCluster().getMetadata())
```

Comment:
Here's an example where you had to swap the order of withRow and withClusterMetadata. This shouldn't be required with builder class. Otherwise we could just have a factory method that takes all necessary arguments.